membership_functions:
  error:
    NB: [-10.0, -8.0, -6.0, -11.0, -8.5, -5.5]
    NS: [-7.0, -4.0, -1.0, -8.0, -4.5, 0.0]
    ZE: [-1.0, 0.0, 1.0, -2.0, 0.0, 2.0]
    PS: [1.0, 4.0, 7.0, 0.0, 4.5, 8.0]
    PB: [6.0, 8.0, 10.0, 5.5, 8.5, 11.0]
  dError:
    DN: [-5.0, -3.0, 0.0, -6.0, -3.5, 1.0]
    DZ: [-1.0, 0.0, 1.0, -1.5, 0.0, 1.5]
    DP: [0.0, 3.0, 5.0, -1.0, 3.0, 6.0]
  wind:
    SNW: [-10.0, -8.0, -5.0, -11.0, -8.5, -4.5]
    WNW: [-7.0, -4.0, -1.0, -8.0, -4.5, 0.0]
    NWN: [-1.0, 0.0, 1.0, -1.5, 0.0, 1.5]
    WPW: [1.0, 4.0, 7.0, 0.0, 4.5, 8.0]
    SPW: [6.0, 8.0, 10.0, 5.5, 8.5, 11.0]
  correction:
    XLNC: [-7.5, -6.0, -4.5, -8.5, -6.5, -4.0]
    LNC: [-5.0, -4.0, -2.5, -5.5, -4.5, -2.0]
    SNC: [-3.0, -1.5, 0.0, -3.5, -1.5, 0.5]
    NC: [-0.5, 0.0, 0.5, -1.0, 0.0, 1.0]
    SPC: [0.0, 1.5, 3.0, -0.5, 1.5, 3.5]
    LPC: [2.5, 4.0, 5.0, 2.0, 4.5, 5.5]
    XLPC: [4.5, 6.0, 7.5, 4.0, 6.5, 8.5]
rules:
  - [PB, DZ, NWN, LPC]
  - [PS, DZ, NWN, SPC]
  - [ZE, DZ, NWN, NC]
  - [NS, DZ, NWN, SNC]
  - [NB, DZ, NWN, LNC]
  - [PB, DN, NWN, LPC]
  - [PS, DN, NWN, SPC]
  - [ZE, DN, NWN, SNC]
  - [NS, DN, NWN, NC]
  - [NB, DN, NWN, SNC]
  - [PB, DP, NWN, NC]
  - [PS, DP, NWN, SNC]
  - [ZE, DP, NWN, SPC]
  - [NS, DP, NWN, SNC]
  - [NB, DP, NWN, LNC]
  - [ZE, DZ, SPW, LNC]
  - [ZE, DZ, SNW, LPC]
  - [ZE, DZ, WPW, SNC]
  - [ZE, DZ, WNW, SPC]
  - [PB, DZ, SPW, XLPC]
  - [NB, DZ, SNW, XLNC]
